Design.md

```diff
@@ -0,0 +1,96 @@
+1. A UInt16 band indicating the frame number the cumulative displacement value each pixel is sourced from
+2. A sidecar metadata file with the same information as the cumulative displacement sidecar files, except for secondary date and OPERA frame image URL
+
+**Note: a time-series stack of OPERA-DISP products from the same frame are not guaranteed to have the same reference date or location!**
+
+However having the same spatiotemporal reference for each pixel in a time-series stack is an important requirement for date/location re-referencing and velocity calcultions. Thus, we standardize the spatiotemporal reference for each product before mosaicking. 
+
+Note that we only standardize the spatiotemporal reference so that each location is consistent through time. We do not standardize this information across space. Non-continuous frame coverage, and varying data collection dates makes standardizing across space a difficult challenge that is planned to be undertaken during the creation of the OPERA-DISP Vertical Land Motion project.
+
+### 1x1 Degree Tile Creation
+Prior to mosaicking of OPERA-DISP products, we generate the metadata rasters.
+
+For each 1x1 degree tile, there is one metadata raster for each orbit direction, for a total of two per tile. The metadata rasters will each span a 1x1 degree area in the web mercator (EPSG:3857) projection with a 30 m pixel spacing. The frame band is created by cross-referencing each tile's extent with the OPERA-DISP frame footprints. A geospatial representation of the OPERA-DISP frame footprints can be created using the [OPERA burst_db utility](https://github.com/opera-adt/burst_db). Frames will be overlaid so that frames from the same relative orbit are contiguous. Relative orbits are stacked from East to West on the ascending pass (West on top), and from West to East on the descending pass (East on top). Within a relative orbit, frames are overlaid so that older frames are on top of younger frames. This corresponds to North on top for the ascending pass, and South on top for the descending pass.
+
+Once the frame band has been constructed, we set the spatiotemporal reference metadata using the values present in the earliest product available for each frame.
+
+We then use the generated metadata rasters as a guide for creating all 1x1 degree cumulative displacement tiles in a time-series stack. The procedure is as follows:
+
+- Specify a date range you want to generate a tile for
+- Generate a new raster with the same projection, resolution, and extent as the metadata raster with three bands
+    - Cumulative displacement
+    - Days since 2014/01/01
```

Comment:
The intent here is to document the secondary date for each pixel and "days since" let's us express that as an int (a valid datatype for a geotiff band), rather than a date or string?

If so, Unix timestamp feels like a more standardized way of representing a date as an integer.

Reply:
Do we need to document reference data per pixel as well?